.devcontainers/devcontainer.json

```diff
@@ -0,0 +1,41 @@
+// For format details, see https:https://containers.dev/implementors/spec/. For config options, see the README at:
+// https://github.com/devcontainers/images/tree/main/src/base-debian
+{
+    "name": "Kubernetes with e2e testing image build and push",
+    "image": "mcr.microsoft.com/devcontainers/base:bullseye",
```

Comment:
Is this something we'll need to bump in the future? If so, under what circumstances?

Reply:
Should we be using kube-cross here? (`registry.k8s.io/build-image/kube-cross`)

Reply:
Looking at https://containers.dev/guide/dockerfile, it looks like we could possibly reference the `../build/build-image/Dockerfile`?

Reply:
maybe something like:
```
    "build": {
        "dockerfile": "../build/build-image/Dockerfile",
        "args": {
            "KUBE_CROSS_IMAGE": "registry.k8s.io/build-image/kube-cross",
            "KUBE_CROSS_VERSION": "v1.29.0-go1.21.3-bullseye.0"
        }
    },
```

Reply:
I think this is a good idea, and something we could use.